libraries/bot-azure/src/test/java/com/microsoft/bot/azure/AzureQueueTests.java

```diff
@@ -97,28 +105,31 @@ public void continueConversationLaterTests() {
 
         QueueMessageItem messages = queue.receiveMessage();
         JacksonAdapter jacksonAdapter = new JacksonAdapter();
-        String messageJson = messages.getMessageText();
+        String messageJson = new String(Base64.decodeBase64(messages.getMessageText()));
         Activity activity = null;
+
         try {
             activity = jacksonAdapter.deserialize(messageJson, Activity.class);
         } catch (IOException e) {
             e.printStackTrace();
             Assert.fail();
         }
-        Assert.assertTrue(activity.isType(ActivityTypes.EVENT));
+
+        Assert.assertEquals(ActivityTypes.EVENT, activity.getType());
         Assert.assertEquals(ActivityEventNames.CONTINUE_CONVERSATION, activity.getName());
         Assert.assertEquals("foo", activity.getValue());
         Assert.assertNotNull(activity.getRelatesTo());
+        activity.getConversation().getProperties().remove("group");
```

Comment:
I don't remember if we still need this:
- If yes: comment it
- If nope: remove it

Reply:
Removed

Reply:
Awesome